migrate/data/migrations/1701872471_initialise_schema.up.sql

```diff
@@ -91,3 +91,9 @@ CREATE TABLE observation (
 
 	PRIMARY KEY (ts_id, obstime_instant)
 );
+
+CREATE USER db_user WITH NOSUPERUSER NOCREATEDB NOCREATEROLE PASSWORD 'mysecretpassword';
+ALTER ROLE db_user SET search_path TO public;
+GRANT USAGE ON SCHEMA public TO db_user;
+GRANT SELECT, INSERT, UPDATE, DELETE ON ALL TABLES IN SCHEMA public TO db_user;
+GRANT USAGE, SELECT ON ALL SEQUENCES IN SCHEMA public to db_user;
```

Comment:
What does adding `db_user` get us?
It is also confusing that we now have two users with the same password.

Reply:
At the moment nothing. In the future, I think it would be good practice to have a user with fewer privileges than the super user to interact with the database. What would you propose, that I remove the user, or change the password?

Reply:
Removed the user.